packages/frontend/src/features/metricsCatalog/components/MetricPeekModal.tsx

```diff
@@ -94,53 +94,82 @@ export const MetricPeekModal: FC<Props> = ({ opened, onClose }) => {
         <Modal.Root
             opened={opened}
             onClose={handleClose}
-            yOffset={150}
+            yOffset={100}
             scrollAreaComponent={undefined}
-            size="80%"
+            size="auto"
         >
             <Modal.Overlay />
-            <Modal.Content sx={{ overflow: 'hidden' }} radius="md">
+            <Modal.Content sx={{ overflow: 'hidden' }} radius="lg" w="100%">
                 <LoadingOverlay
                     visible={
                         metricQuery.isLoading || metricResultsQuery.isLoading
                     }
                 />
                 <Modal.Header
                     sx={(theme) => ({
-                        borderBottom: `1px solid ${theme.colors.gray[4]}`,
+                        borderBottom: `1px solid ${theme.colors.gray[2]}`,
+                        padding: `${theme.spacing.md} ${theme.spacing.lg}`,
                     })}
                 >
                     <Group spacing="xs">
-                        <Hash />
-                        <Text fw={500}>Metric Details</Text>
+                        <Text fw={600} fz="lg" color="dark.7">
+                            {metricQuery.data?.label}
+                        </Text>
+                        <Tooltip
+                            label={metricQuery.data?.description}
+                            disabled={!metricQuery.data?.description}
+                        >
+                            <MantineIcon
+                                color="dark.3"
+                                icon={IconInfoCircle}
+                                size={18}
+                            />
+                        </Tooltip>
                     </Group>
                     <Modal.CloseButton />
                 </Modal.Header>
 
-                <Modal.Body p={0}>
-                    <Flex align="stretch" gap={0}>
-                        <Stack p="md" bg="gray.0" w={360} spacing="sm">
+                <Modal.Body
+                    p={0}
+                    h="100%"
+                    sx={{ display: 'flex' }}
+                    miw={800}
+                    mih={600}
+                >
+                    <Stack p="xl" bg="#FDFDFD" h="100%" w={360} spacing="xl">
```

Comment:
this color, should we put it somewhere nice?

Reply:
agree! Will put in separate pr!